app/component/summary-row.scss

```diff
@@ -1,5 +1,5 @@
 .itinerary-summary-row {
-  min-height: 3.3em;
+  min-height: 50px;
```

Comment:
Specifying min-height in px is totally fine, as long as the actual size of the element scales as it does in this case.